cmd/shield/help.go

```diff
@@ -336,6 +336,9 @@ func ShowHelp(command string) {
 		fmt.Printf("                  in days (7d) or weeks (5w).\n")
 		fmt.Printf("                  This field is @W{required}.\n")
 		fmt.Printf("\n")
+		fmt.Printf("  --retries       How many times to retry the backup job if it fails. Can be given\n")
+		fmt.Printf("                  in times (7).\n")
```

Comment:
The default should be in parens, is it 7 or is it 3 as the code below?

Reply:
I'll change this to 3.